src/plugins/dashboard/public/dashboard_app/no_data/dashboard_app_no_data.tsx

```diff
@@ -38,11 +38,7 @@ function generateId() {
   return uuidv4();
 }
 
-export const DashboardAppNoDataPage = ({
-  onDataViewCreated,
-}: {
-  onDataViewCreated: () => void;
-}) => {
+export const DashboardAppNoDataPage = ({ showPage }: { showPage: () => void }) => {
```

Comment:
**nit** Personally, the `showPage` naming is a bit confusing for me 😅 Since it's always setting it to `false`. Maybe `hideNoDataPage` would be more descriptive?

Reply:
resolved with e2cf5d2dbc707a0164fcc024724e1b0577644f73